tools/ot/src/main/java/dk/alexandra/fresco/tools/ot/otextension/Rot.java

```diff
@@ -0,0 +1,21 @@
+package dk.alexandra.fresco.tools.ot.otextension;
+
+/**
+ * Factory for getting a protocol instance of random OT extension.
+ */
+public interface Rot {
```

Comment:
The usages of this class is purely AbstractFactory.

I suggest renaming the class RotFactory, and have the methods be called create rather than get.

Reply:
Changed. I also did the same for Cote. Commit 27148188